recipes/pyshtools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pyshtools" %}
+{% set version = "4.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd9d6ebb8f6e04cfc7f5989a21c27ede5fc781252f2ef65ae1acd9203c94a30d
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - fftw >=3
+    - liblapack
+  run:
+    - python
+    - numpy
+    - scipy >=0.14.0
+    - matplotlib-base
+    - astropy
+    - xarray
+    - fftw >=3
+    - requests
+
+test:
+  imports:
+    - pyshtools
+
+about:
+  home: https://github.com/SHTOOLS/SHTOOLS
+  license: BSD-3-Clause
```

Comment:
Just noticed this. I wonder if SHTOOLS should be GPL licensed since fftw is GPL and it seems this package links against fftw.

https://en.wikipedia.org/wiki/GPL_linking_exception